ui/site/src/powertip.ts

```diff
@@ -64,7 +64,7 @@ const imagePowertip = (el: HTMLElement) =>
     });
 
 function powerTipWith(el: HTMLElement, ev: Event, f: (el: HTMLElement) => void) {
-  if ('ontouchstart' in window && !el.classList.contains('mobile-powertip')) return;
+  if ('ontouchstart' in window && !el.classList.contains('challenge-powertip')) return;
```

Comment:
the concept of `challenge` only belongs in `lobby` and not in `site` which is more general.